samza-kafka/src/main/scala/org/apache/samza/config/KafkaConfig.scala

```diff
@@ -259,7 +259,22 @@ class KafkaConfig(config: Config) extends ScalaMapConfig(config) {
     //  kafkaChangeLogProperties.setProperty("cleanup.policy", "compact,delete")
     //  kafkaChangeLogProperties.setProperty("retention.ms", String.valueOf(KafkaConfig.DEFAULT_RETENTION_MS_FOR_BATCH))
     // }
-    kafkaChangeLogProperties.setProperty("cleanup.policy", "compact")
+
```

Comment:
Minor: I think you can remove the code block above and just leave the comment with the ticket.

Reply:
The "compact,delete" is only available on 0.11.1.57 or later, which is fairly new. We will stay with "delete" for now.